doc/lua_api.txt

```diff
@@ -2211,7 +2212,80 @@ Elements
 **Note**: do _not_ use a element name starting with `key_`; those names are
 reserved to pass key press events to formspec!
 
+### `style[<name>;<prop1>;<prop2>;...]`
 
+Set the style for the named element `name`.
+Note: this **must** be before the element's tag.
+
+See [Styling Formspecs].
+
+
+### `style_type[<type>;<prop1>;<prop2>;...]`
+
+Sets the style for all elements of type `type` which appear after this tag.
```

Comment:
`tag` -> `element`
Even though it's not a visual one, it's the term for formspec parts.